projects/components/src/navigation/navigation-list.component.ts

```diff
@@ -95,6 +130,18 @@ export class NavigationListComponent implements OnChanges {
     return this.collapsed ? IconType.TriangleRight : IconType.TriangleLeft;
   }
 
+  public onNavItemSelected(navItemLink: NavItemLinkConfig): void {
+    if (this.usePageLevelTimeRange && !isNil(navItemLink.timeRange)) {
+      if (navItemLink.timeRange?.isCustom()) {
```

Comment:
This is a great example of why isCustom never really got used. What you're trying to determine is - is this relative or fixed? That's slightly different than is it custom, which doesn't really have meaning in our current UI. I would explicitly check instanceof Relative instead.